lib/msal-browser/src/custom_auth/core/auth_flow/AuthFlowErrorBase.ts

```
@@ -153,11 +153,9 @@ export abstract class AuthFlowErrorBase {
     protected isVerificationContactBlockedError(): boolean {
         return (
             this.errorData instanceof CustomAuthApiError &&
-            this.errorData.error === CustomAuthApiErrorCode.INVALID_REQUEST &&
-            this.errorData.errorCodes?.includes(550024) === true &&
-            this.errorData.errorDescription?.includes(
-                "multi-factor authentication method is blocked"
-            ) === true
+            this.errorData.error === CustomAuthApiErrorCode.ACCESS_DENIED &&
```

I think you can just use the suberror field and ignore the error field

---

Thanks Danilo. IMO, I will suggest using both error and sub-error to detect this error type in case the sub-error will be used for the other scenarios in the future. If you have strong opinion, please let me know.